packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,13 +10,16 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
-
 -   [*] Update loading and failed screens for web version of the editor [#32395]
-## 1.55.2
 -   [*] Handle floating keyboard case - Fix issue with the block selector on iPad. [#33089]
--   [**] Fix incorrect block insertion point after blurring the post title field. [#32831]
+-   [*] Embed Block Bottom Sheet to set a link. [#31343]
+
+## 1.56.0
```

Comment:
I see that there are more changes in the [`1.56.0` GB-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3664) but they're not reflected here 🤔 .

Reply:
That's because those were added directly on the GBM PRs and not in the `Gutenberg` ones 😅

Reply:
Oh ok, thanks for info 🙇 ! Sometimes it's hard to know what landed in each place 😄 .